src/include/duckdb/execution/operator/projection/OrdinalityData.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
The name of this file should be `ordinality_data.hpp`, I'm also not sure this is where it should live